nvflare/edge/aggregators/edge_result_accumulator.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
     def accept(self, shareable: Shareable, fl_ctx: FLContext) -> bool:
         self.log_info(fl_ctx, f"Accepting: {shareable}")
 
-        w = shareable.get(MsgKey.PAYLOAD)
+        w = shareable.get("weights")
```

Comment:
I think I prefer a fixed key for easier alignment among executor, aggregator, and processor for simulator, but that's a minor thing

Reply:
This is just an example. You will need to write a different aggregator any way.